curriculum/challenges/english/07-scientific-computing-with-python/learn-the-bisection-method-by-finding-the-square-root-of-a-number/65ef1be2b958cabf712296cd.md

```diff
@@ -9,7 +9,9 @@ dashedName: step-11
 
 Now you'll repeatedly narrow down the interval by finding the midpoint of the current interval and comparing the square of the midpoint with the target value.
 
-For that, inside the `else` block, create a `for` loop that runs up to `max_iterations` times.
+For that, inside the `else` block, create a `for` loop that runs up to `max_iterations` times. To iterate you can use range function, it returns a sequence of numbers, starting from 0 by default, and increments by 1 (by default), and stops before a specified number.
```

Comment:
From the instruction, it seems the campers would utilise the `range(start, stop)` variation.
The tests are only valid for `range(stop)`.

You can either modify the instruction to match the test or vice verca.